learning_journal/__init__.py

```diff
@@ -10,12 +13,18 @@
 def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
+    database_url = os.environ.get('DATABASE_URL', None)
+    if database_url is not None:
+        settings['sqlalchemy.url'] = database_url
     engine = engine_from_config(settings, 'sqlalchemy.')
     DBSession.configure(bind=engine)
     Base.metadata.bind = engine
     config = Configurator(settings=settings)
-    config.include('pyramid_chameleon')
+    config.include('pyramid_jinja2')
     config.add_static_view('static', 'static', cache_max_age=3600)
     config.add_route('home', '/')
+    config.add_route('entry', '/entry/{entry}')
+    config.add_route('add_entry', '/write')
+    config.add_route('edit_entry', '/{entry}/edit')
```

Comment:
Think a bit about the logic of the URLs your users will see.  As things stand, you have three different url paths, but you could do the same thing with a more unified URL layout:

```
/entry/{id}
/entry/write
/entry/{id}/edit
```